pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -241,3 +323,66 @@ func New(_ runtime.Object, handle framework.Handle, fts feature.Features) (frame
 		enableReadWriteOncePod: fts.EnableReadWriteOncePod,
 	}, nil
 }
+
+// canBePreempt only verifies the volume, not the resource/affinity..
+func canBePreempt(nodeInfo *framework.NodeInfo, key string) bool {
```

Comment:
How does the preemption logic account for pod priorities?

Reply:
this is unused, right?

Reply:
Sorry I will delete it. Preemption is neither used nor implemented, now only the scheduling problem is considered